doc/sphinx/source/howto/execute.rst

```diff
@@ -34,42 +34,37 @@ Define the computation
 ======================
 
 To a :term:`framework`, a computation is simply a transformation of inputs to 
-outputs. While a *framework bridge* can programmatically construct the graph 
+outputs. While a :term:`bridge` can programmatically construct the graph 
 from a framework's representation of the computation, graph construction can be 
 somewhat more tedious for users. To a user, who is usually interested in 
```

Comment:
I think "users" is the wrong word here, since this isn't the expected use. Maybe something along the lines of "when done directly" rather than "for users". These sections are aimed at people developing their own bridge, or, even, their own framework.

Reply:
Yes, agreed. Updated this section.